puppet/modules/slave/manifests/unittests.pp

```diff
@@ -3,6 +3,20 @@
   Stdlib::Absolutepath $homedir,
 ) {
   $is_el8 = $facts['os']['family'] == 'RedHat' and $facts['os']['release']['major'] == '8'
+  $is_el9 = $facts['os']['family'] == 'RedHat' and $facts['os']['release']['major'] == '9'
+
+  if $is_el9 {
+    if $facts['os']['name'] == 'CentOS' {
+      yumrepo { 'crb':
+        enabled => '1',
+      }
+    }
+    if $facts['os']['name'] == 'RedHat' {
```

Comment:
Should this be `case $facts['os']['name']`? And I'll admit that I still often look up https://www.puppet.com/docs/puppet/7/lang_conditional.html for the exact syntax.

Reply:
Should it be? No idea. Could it be? Probably.

Reply:
I think it should, because you're essentially doing the same thing. So you could make it determine `$crb_repo_name` and then have a single `yumrepo` statement.